bedrock/newsletter/templates/newsletter/existing.html

```diff
@@ -19,7 +19,6 @@
     {% if did_confirm %}
       <h1>{{ _('Thanks for Subscribing!') }}</h1>
 
-      {% if l10n_has_tag('newsletter_confirm_2016') %}
```

Comment:
I checked and all of the langfiles for this page have this tag. Removing the old strings.